Gradle : running a single test ?

not a real issue: I'm trying to run a **single test** with gradle but i cannot find the correct command-line. I've tried : 

`./gradlew -Dtest.single=htsjdk.variant.vcf.VCFEncoderTest test`

but it doesn't work. (is it possible to run such test ?)

Can someone please update the wiki https://github.com/samtools/htsjdk/wiki/Writing-tests-in-HTSJDK with the instruction to run a single test ?

thanks !

---

Why ? I'm working behind a strong firewall, it takes a very long time before gradle raises a ' java.net.ConnectException' for the tests requiring a connection.

```
ParsingUtilsTest:
[TestNG] Running:
  Command line suite

- testFTPDoesExist *** FAILED ***
  java.lang.AssertionError: expected [true] but found [false]
  at org.testng.Assert.fail(Assert.java:94)
  at org.testng.Assert.failNotEquals(Assert.java:496)
  at org.testng.Assert.assertEquals(Assert.java:125)
  at org.testng.Assert.assertEquals(Assert.java:288)

```

--- a/README.md
+++ b/README.md
@@ -49,12 +49,9 @@ Example gradle usage from the htsjdk root directory:
  ```
  ./gradlew test
 
- ./gradlew test -Dtest.single=TestClassName
+ ./gradlew test --tests AlleleUnitTest
 
- ./gradlew test --tests htsjdk.variant.variantcontext.AlleleUnitTest
- ./gradlew test --tests "*AlleleUnitTest"
-
- ./gradlew test --tests "*AlleleUnitTest" --debug-jvm
+ ./gradlew test --tests AlleleUnitTest --debug-jvm
  ```
 
 - run tests and collect coverage information (report will be in `build/reports/jacoco/test/html/index.html`)

